src/main/java/ladder/formatter/PlayersPrintFormatter.java

```diff
@@ -0,0 +1,18 @@
+package ladder.formatter;
+
+import ladder.domain.Player;
+import ladder.domain.Players;
+
+import java.util.stream.Collectors;
+
+public class PlayersPrintFormatter implements DataPrintFormatter<Players> {
+
+    private final NameFormatter<Player> playerNameFormatter = new PlayerNameFormatter();
+
+    @Override
+    public String dataPrintFormat(Players players) {
+        return players.getPlayers().stream()
+                .map(player -> String.format(playerNameFormatter.nameFormat(player), player.getName()))
```

Comment:
아래의 코드로 개선해 보면 어떨까요?

```suggestion
                .map(player -> String.format("%6s", player.getName()))
```

Reply:
기초적인 것을 잊고 어렵게 구현하고 있었네요... 조언 감사합니다!! 👍 👍